app/frontend/src/components/MarkdownInput/MarkdownInput.test.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  render,
+  screen,
+  waitFor,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { CONFIRM_UPLOAD, SELECT_AN_IMAGE } from "components/constants";
+import {
+  IMAGE_DESCRIPTION,
+  INSERT_IMAGE,
+} from "components/MarkdownInput/constants";
+import { useForm } from "react-hook-form";
+import { service } from "service";
+import wrapper from "test/hookWrapper";
+import { MockedService } from "test/utils";
+
+import MockedMarkdownInput from "./MarkdownInput";
```

Comment:
Does this import alone not work? I thought the mocked input only gets pulled in if you explicitly do `jest.mock("./MarkdownInput")` since this is a relative import so [jest doesn't auto mock them](https://jestjs.io/docs/26.x/manual-mocks#mocking-user-modules)?